src/files/XmlFile.spec.ts

```diff
@@ -377,12 +375,11 @@ describe('XmlFile', () => {
 
     describe('getCompletions', () => {
         it('formats completion paths with proper slashes', () => {
-            let scriptPath = s`C:/app/components/component1/component1.brs`;
-            program.files[scriptPath] = new BrsFile(scriptPath, s`components/component1/component1.brs`, program);
+            program.addOrReplaceFile('pkg:/components/component1/component1.brs', '');
 
-            let xmlFile = new XmlFile(s`${rootDir}/components/component1/component1.xml`, s`components/component1/component1.xml`, <any>program);
+            let xmlFile = new XmlFile(s`${rootDir}/components/component1/component1.xml`, 'pkg:/components/component1/component1.xml', <any>program);
```

Comment:
Should `s` be also removed for the `srcPath` argument, as it is normalised by the file class as well?

Reply:
Yeah, we can remove it. I figured this PR was huge already, and this is in a spec file, so it wasn't overly worthwhile do to that right now. I plan on doing a large refactoring of the test suite sometime after this lands. However, if you really want, I can do all of that refactoring in here too.